crates/cli/commands/src/stage/drop.rs

```diff
@@ -47,18 +48,35 @@ impl<C: ChainSpecParser> Command<C> {
             _ => None,
         };
 
-        // Delete static file segment data before inserting the genesis header below
+        // We can only enqueue the pruning because if the task is interrupted after we have deleted
+        // the files, BUT before we have committed the DB we'd lose essential data and would need to
+        // recover it manually (eg. copy paste static files from other node).
```

Comment:
hmm, not super obvious what this means exactly. unsure what `enqueue` refers to here.

or rather what operation this comment tries to explain